guideline-03.md

```diff
@@ -1,3 +1,5 @@
-**3. A stable version of your plugin must be available from its WordPress Plugin Directory page.**
+<h4>3. A stable version of a plugin must be available from its WordPress Plugin Directory page.</h4>
 
-The only version of the plugin that WordPress.org distributes is the one in the directory. Though you may develop your code somewhere else, please remember that users will be downloading from the directory, not your development environment.
+The only version of the plugin that WordPress.org distributes is the one in the directory. Though people may develop their code somewhere else, users will be downloading from the directory, not the development environment.
+
+Conversely, not using a directory and instead distributing code via alternate methods may result in a plugin being removed.
```

Comment:
Shouldn't this be "Conversely, not using *the* directory"?

Reply:
It means 'a' as in 'the one we just gave you.'  Would "Conversely, not using the provided directory..." be better?

Reply:
I took "directory" to be referring to the WordPress Plugin Directory. I guess it is meant to refer to the SVN repo/directory listing here. Maybe a different term than "directory" would be less ambiguous.

Reply:
Does it matter if it’s a little ambiguous? I mean, the intent is to stop people from submitting a request, getting a review, and then only pushing one version to drive more traffic to their own site. (Or people who get a review and never use the hosting at all... which is a resource drain). Basically ... use what we gave you :)

What would a better term be in that case?

Reply:
I guess the ambiguity doesn't really matter as far as the intent, it just threw me off and distracted me from understanding fully what was meant. Because of the way "Directory" is used in the heading for this section, I also took the first paragraph to be using  the term the same way (to refer to the Plugin Directory). So when I got to the second paragraph, I thought "directory" was still intended to mean that same thing. If this is standard terminology used for plugins on .org, then I guess it is maybe just because I'm not familiar with it; I'd have expected "repo" or something like that to be used to refer to a particular plugin's ~~repo~~ directory.

Reply:
The SVN repository is where you (personally as a developer) update your singular plugin.

The Plugin Directory is wordpress.org/plugins as a whole

Yeah, I totally get where that's confusing 🤦‍♀️

Reply:
Okay, I did a bit of a rewrite to make it more clear! let me know if that helps @JDGrimes

Reply:
Yes, that is much clearer to me. 👍

Reply:
`not using a directory` in the final paragraph seems like it should be `not using the directory`. `the` is used in the preceding paragraph.